src/classes/STG_InstallScript_TEST.cls

```diff
@@ -425,6 +431,32 @@ public with sharing class STG_InstallScript_TEST {
         }
     }
 
+    /*********************************************************************************************************
+    @description 
+        Verify that the Customizable Rollup Settings matches the expected Configuration
+    **********************************************************************************************************/ 
+    private static void assertRollupSettingsEqual(Customizable_Rollup_Settings__c expectedConfig) {
```

Comment:
Is it worth splitting this out since it's only referenced one time?

Reply:
I think if we were to put into the actual push() test function, the basic structure of that test method would be different from the other ones.  I've added the new constants for the other values though.

Reply:
Fair enough. It also makes the test method more readable. I would suggest updating the class to use `@IsTest` and new method names to bring in line with our current standards.

Reply:
Thanks, I've updated the test class.  Can you also review the critical changes section?  Radmila brought up a good point - because we are switching settings - do we need to make our customers aware of this change?